packages/testkit-backend/src/cypher-native-binders.js

```diff
@@ -183,6 +183,14 @@ export default function CypherNativeBinders (neo4j) {
       return structResponse('CypherVector', { dtype, data })
     }
 
+    if (neo4j.isUnsupportedType(x)) {
+      const name = x.name
+      const minimumProtocolMajor = x._minimumProtocolMajor
+      const minimumProtocolMinor = x._minimumProtocolMinor
+      const message = x.message
+      return structResponse('CypherUnsupportedType', { name, minimumProtocol: minimumProtocolMajor + '.' + minimumProtocolMinor, message })
```

Comment:
```suggestion
      const minimumProtocol = x.minimumProtocolVersion
      const message = x.message
      return structResponse('CypherUnsupportedType', { name, minimumProtocol, message })
```
Testing the public API rather than the internal one seems preferable.